db/migrate/20200122175044_remove_column_from_friendships.rb

```diff
@@ -0,0 +1,5 @@
+class RemoveColumnFromFriendships < ActiveRecord::Migration[6.0]
+  def change
+    remove_column :friendships, :confirmed
```

Comment:
Why remove the confirmed column?
It's a requirement ([Screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/rails/final_project/milestone1/status.png) from Pathwiright)